src/engines/html5/capabilities/html5-autoplay.js

```diff
@@ -1,10 +1,17 @@
 // @flow
 import * as Utils from '../../../utils/util'
 import * as EncodingSources from '../../../assets/encoding-sources.json'
+import {Html5EventType} from '../../../event/event-type'
+
+const WAIT_TIME: number = 500;
+
+const testVideoElement: HTMLVideoElement = Utils.Dom.createElement('video');
+testVideoElement.src = EncodingSources.Base64Mp4Source;
+// For iOS devices needs to turn the playsinline attribute on
+testVideoElement.setAttribute('playsinline', 'playsinline');
```

Comment:
playsinline doesn't have a value, just add
`testVideoElement.setAttribute('playsinline', '');`

Reply:
we do the same in html5 engine

Reply:
playsinline is a boolean attribute, so, any value turns it on, using the name of the attribute is an often used convention